backups/backup.py

```diff
@@ -105,10 +106,13 @@ def run_importer_script() -> None:
                 )
     finally:
         if popen:
+            log.info("Before terminate %s", popen.returncode)
             popen.terminate()
+            log.info("After terminate %s", popen.returncode)
 
     if (return_code := popen.returncode) != 0:
-        raise subprocess.CalledProcessError(return_code, command)
+        log.error("Capella run with command %s failed. Exit code: %d", command, return_code)
```

Comment:
What is the benefit of having the logging message here? 
The information is already printed as part of the stacktrace: 
```
subprocess.CalledProcessError: Command '<command>' returned non-zero exit status <exit code>.
```

Reply:
And how does this PR change the behaviour then? The script should still stop as an exception is raised in line 115 on non-zero exit codes.